java-android/hooks/images.csv

```diff
@@ -0,0 +1,3 @@
+DOCKERFILE;ARGS (comma-separated list);TAG
+;OPENJDK=openjdk:10.0.2-jdk-slim;jdk10
+;OPENJDK=openjdk:11.0.7-jdk-slim-buster;jdk11
```

Comment:
AFAIK, current LTS versions of Java are 8 and 10? In this case, maybe replace jdk10 with jdk8 if we want to support it (do we?)

(If it makes sense to support Java 8, but currently it's not supported, we can add jdk8 here and file an issue in roc-java for Java 8 support).

Reply:
Yes actually Java 8 and Java 11 are the two LTS versions. 
We can surely replace jdk10 with jdk8 (currently is already supported by `roc-java`) for extending support on much wider build environment.